packages/vendor-core/scss/loading-state.scss

```diff
@@ -0,0 +1 @@
+@import '~patternfly/dist/sass/patternfly/_loading-state';
```

Comment:
Isn't this file redundant?
I would say, put `@import '~patternfly/dist/sass/patternfly/_loading-state';` in `vendor-core.scss` together with all the other 3rd-party scss files.